crt_portal/cts_forms/model_variables.py

```diff
@@ -438,6 +438,7 @@
     ('3 Year', _('3 Year')),
     ('10 Year', _('10 Year')),
     ('Permanent', _('Permanent')),
+    ('(none)', _('None set')),
```

Comment:
Is this necessary? The value as implemented for an unset schedule is `null` - this will change it to 'None set'. The other fields appear to do this using placeholders. e.g. you can't choose "None set" for user assignment.

e.g.:

![unassign](https://github.com/usdoj-crt/crt-portal/assets/15126660/c9f8aa6d-49b0-4eff-8cc3-15765afd529d)